torchao/ops.py

```diff
@@ -71,6 +72,56 @@ def _(
     return _in_feats.new_empty((BS, OC))
 
 
+def scaled_dot_product_int8(
```

Comment:
please add `for_cpu` to the op name, since we will likely add some gpu ops as well